infra/base-images/base-builder/Dockerfile

```diff
@@ -27,6 +27,27 @@ RUN apt-get install -y git \
     libblocksruntime-dev \
     libc6-dev-i386
 
+# Download and install the latest stable Go.
+ADD https://storage.googleapis.com/golang/getgo/installer_linux $SRC/
+
+# $SHELL is needed for the installer.
+ENV SHELL "bash"
+RUN chmod +x $SRC/installer_linux && \
+    $SRC/installer_linux && \
+    rm $SRC/installer_linux
+ENV SHELL ""
+
+# Set up Golang environment variables (copied from /root/.bash_profile).
+ENV PATH $PATH:/root/.go/bin
```

Comment:
nit: this block can be shortened to
ENV GOPATH /root/go
ENV PATH $PATH:/root/.go/bin:$GOPATH/bin

Also, why do we have /root/go and /root/.go ?